src/Simplex/Chat/Controller.hs

```diff
@@ -466,6 +514,7 @@ data ChatResponse
   | CRMsgIntegrityError {user :: User, msgError :: MsgErrorType}
   | CRCmdAccepted {corr :: CorrId}
   | CRCmdOk {user_ :: Maybe User}
+  | CRCmdRelayed -- ^ A command sent to a remote host instead of being interpreted locally
```

Comment:
why does UI need to know the difference? The response should be synchronous anyway, when it was.